### PR TITLE
NMS-14256: Additional XML Collector docs

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14256

### CODE REVIEW

**Review by dino2gnt — DISMISSED** [review]
Looks good from over here.

File: docs/modules/reference/pages/configuration/performance-data-collection/collectors/xml.adoc [hunk 1]
```diff
@@ -98,6 +105,10 @@ The request element can have the following optional child elements:
 | How often to repeat the request in case of an error.
 | 0
 
+| disable-ssl-verification
+| Do not attempt to verify the name of on the SSL certificate against the `xml-source` URL attribute.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/performance-data-collection/collectors/xml.adoc:109]:
```suggestion
| Do not attempt to verify the name on the SSL certificate against the `xml-source` URL attribute.
```
not sure if you prefer "on" or "of". 50/50 chance.

File: docs/modules/reference/pages/configuration/performance-data-collection/collectors/xml.adoc [hunk 2]
```diff
@@ -115,14 +126,26 @@ The referenced `opennms-nodes.xml` file contains:
 </xml-groups>
 ----
 
+== HTTP headers
+
+If the endpoint being collected requires additional headers, they can be added as part of the `<request>` object.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/performance-data-collection/collectors/xml.adoc:131]:
```suggestion
If the endpoint being collected requires additional headers, you can add them as part of the `<request>` object.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor change. I wish there were a way we could just approve with corrections. It's a typo. :-)

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**